src/main/resources/db/migration/V13__only_one_ytelse_per_klagebehandling.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE klage.klagebehandling
+    ADD COLUMN vedtak_id UUID REFERENCES klage.vedtak (id);
+
+UPDATE klage.klagebehandling k
```

Comment:
Viktig å tenke gjennom hvordan dette vil fungere iom at vi har to pods. (Og i og for seg uansett, iom at ny pod spinnes opp før gammel pod stenges ned). Man gjør database-endringene her, men har samtidig pods gående med gammel Hibernate-annotering. De vil da fortsette å lage koblinger ved å sette klagebehandling_id i vedtak. Så man kan risikere å ha koblinger som ikke er migrert også etter at denne PRen er kjørt ut, vil jeg påstå. (Men det kan godt være det ikke skjer, iom at vi ikke har så mye trafikk..) Hvordan håndtere det? Kjøre ut en ny PR med SET vedtak_id = () på nytt kanskje, + alter table drop column?